### ISSUE TITLE
default location to current location

### ISSUE THREAD

**Issue by mutairibassam** [AbduzZami/open_street_map_search_and_pick#13]
Does the package support default location to be set automatically to user current location?

**Comment by FlorianPix** [comment]
I don't think so, but you can set the center (default location) yourself. Here's an [example](https://github.com/FlorianPix/climbing_diary/blob/main/src/flutter_app/lib/pages/map_page/location_picker.dart) (line 46).

**Comment by mutairibassam** [comment]
Big thanks @FlorianPix. Yah, I noticed and did it from my side. I believe it's still a good feature to be added in the package.

I am willing to do a pr to add a new argument called `userLocation` which accept true/false and defaulted to true.
```dart
OpenStreetMapSearchAndPick(
    /// true or false - default true
    userLocation: true,
})
```
